priv/templates/ct_suite.erl

```diff
@@ -0,0 +1,132 @@
+%%%-------------------------------------------------------------------
+%%% @author {{author_name}} <{{author_email}}>
+%%% @copyright (c) {{copyright_year}} {{author_name}}
+%%% @doc
+%%%     Tests {{name}}
+%%% @end
+%%% -------------------------------------------------------------------
+
+-module({{name}}_SUITE).
+
+%% Common Test headers
+-include_lib("common_test/include/ct.hrl").
+
+%% Include stdlib header to enable ?assert() for readable output
+-include_lib("stdlib/include/assert.hrl").
+
+%% Test server callbacks
+-export([
+    suite/0,
+    all/0,
+    groups/0,
+    init_per_suite/1, end_per_suite/1,
+    init_per_group/2, end_per_group/2,
+    init_per_testcase/2, end_per_testcase/2
+]).
```

Comment:
Any opinions on using  `-compile(export_all).` just for the CT suite as Tristan initially suggested in the issue? I know that out of laziness I often revert to that one just for CT suites as well.

Reply:
I'd vote against export_all. Not only it requires to suppress compiler warning, but also creates "accidental interface" that works in test mode only.

So far I've been happy with just evaluating the code (e.g. using power_shell, https://github.com/WhatsApp/power_shell)

Reply:
It is a test module, it only runs in test mode :). And doesn't really have an interface because it isn't called by anything but the ct runner.

Reply:
That one is a test module, but text message suggests to suppress "export_all warning" for the entire project compiled with test profile.
I agree, export_all for test SUITE makes sense, I just wish it's possible to enable warning suppression it only for test/ folder of the project.

Reply:
Wouldn't `-compile([export_all, nowarn_export_all]).` prevent warnings only in this module? It seems to work well on a small shell test here, but I haven't checked across all versions.

Reply:
@max-au true, but a regular compile would still have the warning and building like a release would then still warn or error so it couldn't go to prod.

But point taken and looks like Fred has a solution.